### PR TITLE
docs: Improve migration instructions

### PR DESCRIPTION
**Pull request by Luc1412**
While testing the migration, with the deb package, I encountered a few problems. These changes should improve the migration flow and make a few things more clear.

## What changed?
- The configs should be moved, before installing edumfa. Otherwise it will cause issues with deb package installation.
- The rename instructions where a bit broad. I made it more clear, what has to be renamed.
- I added a hint to not accept the new config during the edumfa installation, otherwise it may cause issues
- It wasn't pretty clear how to call the migration binary. I fixed that

### CODE REVIEW

File: READ_BEFORE_UPDATE.md [hunk 1]
```diff
@@ -16,13 +16,18 @@
 * docs: start fixing incorrect camel case by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/25
 * docs: cleanup old refs and fix path names by @j-hoff in https://github.com/eduMFA/eduMFA/pull/28
 * docs: use fu repo by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/33
-* docs: add docu for new sms provider "Http Message To Uid Provider" by @j-hoff in https://github.com/eduMFA/eduMFA/pull/21
-* feat: smstoken: add configurable URL which is called after check, extend test by @j-hoff in https://github.com/eduMFA/eduMFA/pull/18
-* feat: make result value available in logginghandler, verbosely log set tokeninfo by @j-hoff in https://github.com/eduMFA/eduMFA/pull/19
-* feat: new API endpoint POST /info/<serial> to bulk modify tokeninfo by @j-hoff in https://github.com/eduMFA/eduMFA/pull/20
+* docs: add docu for new sms provider "Http Message To Uid Provider" by @j-hoff
+  in https://github.com/eduMFA/eduMFA/pull/21
```
Comment (fritterhoff) [inline READ_BEFORE_UPDATE.md]: Is there a reason for the new line breaks? (Some formatting anywhere that requires a specific line length? 🤔 )

Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: This is the automated code format from Jetbrain IDEs

Comment (fritterhoff) [reply READ_BEFORE_UPDATE.md]: Would you mind to change your settings? There should be a setting for line breaks. Would prefer to keep the changes in existing changelogs as small as possible,

File: READ_BEFORE_UPDATE.md [hunk 2]
```diff
@@ -51,26 +56,33 @@
 ### Migrating from privacyIDEA 3.9.2 to eduMFA 1.0.0
```
Comment (fritterhoff) [inline READ_BEFORE_UPDATE.md]: Maybe it makes sense to shift this block to the top of the release notes and make it more generic? Migrating from privacyIDEA 3.9.2 to eduMFA

**Comment by Luc1412** [comment]
@fritterhoff Solved the line wrapping

**Review by fritterhoff — APPROVED** [review]

**Comment by fritterhoff** [comment]
@j-hoff since I changed some lines we need your approval. 😬

File: READ_BEFORE_UPDATE.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+## Migrating from privacyIDEA 3.9.2 to eduMFA
+
+> [!CAUTION]
+>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
+     the database.
+>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
+> 
+> These migration steps are applicable for the versions 1.0.0 to 1.5.1
+
+* Uninstall privacyIDEA and stop your nginx or `Apache2` or `nginx` service
```
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]:
```suggestion
* Uninstall privacyIDEA and stop your `Apache2` or `nginx` service
```

File: READ_BEFORE_UPDATE.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+* Move/Copy all configurations from `/etc/privacyidea/` to `/etc/edumfa/`
+    * The config file needs to be renamed from `pi.cfg` to `edumfa.cfg`
+    * Rename/Replace `PI_` in the configuration file with `EDUMFA_` (e.g. `PI_ENCFILE` changes to `EDUMFA_ENCFILE`)
+    * Update the paths in the configuration file to `/etc/edumfa/`
+    * Update the log file path to `/var/log/edumfa/edumfa.log`
+* Install eduMFA (e.g. using Container, PIP or the `.deb` Package)
+    * When using the `.deb` package, make sure to not overwrite the existing configuration file
```
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: how? will the config be overwritten in any case?

File: READ_BEFORE_UPDATE.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), all following steps will be performed automatically
```
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: I believe crontab or systemd services will not be updated by the .deb packages

File: READ_BEFORE_UPDATE.md [hunk 6]
```diff
@@ -1,5 +1,38 @@
 # Update Notes
 
+## Migrating from privacyIDEA 3.9.2 to eduMFA
+
+> [!CAUTION]
+>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
+     the database.
+>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
+> 
+> These migration steps are applicable for the versions 1.0.0 to 1.5.1
```
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: maybe we change this line until we release a version for which the instructions do NOT work
```suggestion
> These migration steps are applicable for all versions released up to now
```

**Review by j-hoff — CHANGES_REQUESTED** [review]
some minor remarks

File: READ_BEFORE_UPDATE.md (hunk 4 above)
Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: When having a config present, during installation you'll be asked if you like to keep the current config or use the maintainers version.

File: READ_BEFORE_UPDATE.md (hunk 5 above)
Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: The packages adds a new cron, however systemd keeps unchanged. Maybe we should rather mention, what will be done automatically like: "When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will executed automatically. Additionally the web server configurtion and a cron will also be installed.

Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]:
```suggestion
    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will be executed automatically. Additionally the web server configuration and a cron entry will be installed.
```

File: READ_BEFORE_UPDATE.md (hunk 4 above)
Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]:
```suggestion
    * When using the `.deb` package, make sure to not overwrite the existing configuration file (When asked by dpkg, answer N or O to keep your installed version)
```

Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: I think the current explanation should be mostly sufficient. Maybe we can add `during installation` and/or `when prompted`

Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]: OK
```suggestion
    * When using the `.deb` package, make sure not to overwrite the existing configuration file when prompted
```

**Review by j-hoff — APPROVED** [review]

**Pull request merged**